packages/wonder-blocks-cell/src/components/internal/cell-core.js

```diff
@@ -30,48 +30,59 @@ type CellCoreProps = {|
 const CellCore = (props: CellCoreProps): React.Node => {
     const {
         children,
+        horizontalRule = "inset",
         leftAccessory = undefined,
         leftAccessoryStyle = undefined,
         rightAccessory = undefined,
         rightAccessoryStyle = undefined,
+        style,
+        testId,
     } = props;
 
+    const maybeRenderLeftAccessory = (): React.Node => {
```

Comment:
Moved it to its own fn to make it more readable (same with rightAccessory).

Reply:
Minor nit, but you could make these `const LeftAccessory = ...` and then do `<LeftAccessory />` down below. I feel like that tends to make more sense for cases where you're just rendering React nodes anyway!

Reply:
And if you wanted to go a step further you could pass those props into the component and then hoist the accessories up so that you can test then individually, potentially!

Reply:
Hmmm... https://khanacademy.atlassian.net/browse/MOB-4372

Reply:
Fair enough. I used that approach at some point, but changed it to match the Mobile implementation. I'm glad that @lillialexis plans to make that change on the mobile side :).